shell/platform/windows/flutter_windows_view.cc

```diff
@@ -332,6 +340,7 @@ void FlutterWindowsView::SendPointerMove(double x,
   FlutterPointerEvent event = {};
   event.x = x;
   event.y = y;
+  event.view_id = _viewIdForPointerEvent();
```

Comment:
Instead of setting `view_id` in all pointer methods, please set it in the common `SendPointerEventWithData` helper.